Apps/WebClient/src/ClientApp/src/views/profile.vue

```diff
@@ -387,6 +400,7 @@ export default class ProfileView extends Vue {
             })
             .finally(() => {
                 this.isLoading = false;
+                this.showCheckEmailAlert = true;
```

Comment:
this will show even if it fails. Since it needs to show as a notification when the user gets a new email let's move it to the then() section

Reply:
Sounds good! Thanks @FuriousLlama 
Code updated.